tests/unit_tests/blas/level1/rotg_usm.cpp

```diff
@@ -73,6 +73,8 @@ int test(const device &dev) {
     c     = rand_scalar<fp_scalar>();
     a_ref = a;
     b_ref = b;
+    s_ref = s;
+    c_ref = c;
```

Comment:
Do you use these new variables?

Reply:
Yes, they are used. They were not initialized before. I am fixing that here.